cli/module_generate/scripts/generate_stubs.go

```diff
@@ -145,15 +171,31 @@ func parseFunctionSignature(resourceSubtype, resourceSubtypePascal string, funcD
 		return
 	}
 
+	// Receiver
+	receiver = modelType
+	if funcDecl.Recv != nil && len(funcDecl.Recv.List) > 0 {
+		field := funcDecl.Recv.List[0]
+		if f, ok := field.Type.(*ast.StarExpr); ok {
```

Comment:
optional - could we use a more descriptive name for f like pointerType? if I am understanding correctly, f would be assigned to the AST node of type *ast.StarExpr which is a pointer to the receiver type...just an idea to help w clarity